src/integrations/twitch-api/api.ts

```diff
@@ -10,13 +12,15 @@ export class TwitchAPI {
   private twitchAPIWebhookEndpoint: string = `${this.twitchAPIEndpoint}/webhooks/hub`
 
   private headers: object
+  private webhookSecret: string
 
   constructor(private config: Config) {
     this.headers = {
       Authorization: `Bearer ${this.config.twitchChannelAuthToken}`,
       'Content-Type': 'application/json',
       'Client-ID': this.config.twitchClientId
     }
+    this.webhookSecret = Guid.create().toString();
```

Comment:
We're going to have to re-register the webhooks daily. That means we'll have to call `registerWebhooks` daily.  I'm thinking we should move this to the first line inside `registerWebhooks` so that we change the secret every time we have to register.

Reply:
So logical. Coming right up.